apiv2/api/proto/resources/localaccount/v1/localaccount.proto

```diff
@@ -38,8 +38,15 @@ message LocalAccountResource {
     (google.api.field_behavior) = REQUIRED
   ];
   
-  // Timestamp for the creation of the resource.
-  string created_at = 50100 [(google.api.field_behavior) = OUTPUT_ONLY];
-  // Timestamp for the last update of the resource.
-  string updated_at = 50101 [(google.api.field_behavior) = OUTPUT_ONLY];
+  // The local account resource's unique identifier. Alias of resourceId.
+  string local_accountID = 5001 [
```

Comment:
why is this added?

Reply:
Backward compatibility 
https://github.com/open-edge-platform/infra-core/blob/main/api/api/openapi/schemas/localaccounts.yaml#L17